plugin.json

```diff
@@ -34,6 +34,12 @@
                 "type": "bool",
                 "help_text": "(Experimental) When true, Jitsi video is embedded as a floating window inside Mattermost by default. Users can override this setting with '/jitsi settings'."
             },
+            {
+                "key": "JitsiPrejoinPage",
+                "display_name": "Show pre-join page:",
+                "type": "bool",
+                "help_text": "When false, pre-join page will not be displayed when Jitsi is embedded inside Mattermost."
```

Comment:
I would think that this would be a per-user setting

Reply:
@mickmister We can create a per-user setting for this. What should be the UI for this, we can have a slash command `/jitsi settings` and it can post a slack attachment with buttons "Yes" and "No" or is there some other approach in your mind?

Reply:
`/jitsi settings pre-join true/false` should work fine

Reply:
@mickmister Added the slash command